src/scss/jooa11y.scss

```diff
@@ -127,6 +122,7 @@ html[data-jooa11y-theme="dark"] {
   font-weight: normal;
   letter-spacing: unset;
   word-break: normal;
+  -webkit-padding-start: 0;
```

Comment:
not needed - padding-inline-start is fully supported without prefix https://caniuse.com/?search=padding-inline-start